packages/@tailwindcss-upgrade/src/codemods/migrate-border-compatibility.test.ts

```diff
@@ -156,6 +221,7 @@ it('should add the compatibility CSS after the last import, even if a body-less
         border-color: var(--color-gray-200, currentColor);
       }
     }
+
```

Comment:
The whitespace handling in this file a little bit annoying but this is a test for the codemod itself not the full integration test. In integration tests the whitespace is more correct.

This issue existed before, but wanted to point it out explicitly that it's not directly related to this change.